python-backend/app/api/mines/mine/models/mine_identity.py

```diff
@@ -79,6 +76,17 @@ def json(self):
             'mine_type': [item.json() for item in self.active(self.mine_type)]
         }
 
+    def json_for_list(self):
+        return {
+            'guid': str(self.mine_guid),
+            'mine_detail': [item.json() for item in self.mine_detail],
+            'mine_permit': [item.json() for item in self.mine_permit],
+            'mine_status': [item.json() for item in self.mine_status],
+            'mine_tailings_storage_facility':
+            [item.json() for item in self.mine_tailings_storage_facilities],
+            'mine_type': [item.json() for item in self.active(self.mine_type)]
```

Comment:
Totally fair and expected, which is why I approved the changes anyway. Just wanted to mention it since I've been working on that section of the codebase quite a bit lately